LICENSES/Apache-2.0.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I am not sure but i think you added too many licenses. The licenses folder should be use only if you have different licenses in the repo for example Apache and LGPL etc..

Reply:
The LICENCES directory was generated by `reuse init`. Unfortunately `reuse lint` only works with the files under LICENSES and ignores the top level LICENSE file